src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php

```diff
@@ -53,6 +53,7 @@
 use PrestaShop\PrestaShop\Core\Grid\Definition\Factory\CategoryGridDefinitionFactory;
 use PrestaShop\PrestaShop\Core\Search\Filters\CategoryFilters;
 use PrestaShopBundle\Component\CsvResponse;
+use PrestaShopBundle\Controller\Admin\CommonController;
```

Comment:
I don't think we need the use statement here, just for the deprecation comment
I'm even surprised CSFixer doesn't shout at this but it's probably because it takes the comment into account, anyway this is probably overkill but not worth blocking the PR either